### PR TITLE
feat: Convert evaluateVariable function

### PR DESCRIPTION
**Pull request by polok**

### CODE REVIEW

File: Sources/FeaturevisorSDK/Instance+Feature.swift [hunk 1]
```diff
@@ -35,6 +35,49 @@ extension FeaturevisorInstance {
...
+
+    func getMatchedAllocation(
+            traffic: Traffic,
+            bucketValue: Int) -> Allocation? {
+
+        return traffic.allocation.first(where: { allocation in
+            let start = allocation.range.start
+            let end = allocation.range.end
+
+            return start <= bucketValue && end >= bucketValue
```
Comment (fahad19) [inline Sources/FeaturevisorSDK/Instance+Feature.swift:47]: 👍

File: Sources/FeaturevisorSDK/Instance+Feature.swift [hunk 2]
```diff
@@ -35,6 +35,49 @@ extension FeaturevisorInstance {
             return true;
         });
     }
+
+    func getMatchedAllocation(
+            traffic: Traffic,
+            bucketValue: Int) -> Allocation? {
```
Comment (fahad19) [inline Sources/FeaturevisorSDK/Instance+Feature.swift:41]: out of curiosity, is SwiftFormat automatically putting the closing `)` in the same line?

**Review by fahad19 — APPROVED** [review]
🚀

File: Sources/FeaturevisorSDK/Instance+Feature.swift (hunk 2 above)
Comment (polok) [reply Sources/FeaturevisorSDK/Instance+Feature.swift:41]: Actually, I would need to check the defult one. For years I stuck to https://github.com/nicklockwood/SwiftFormat and the current rules that we have in my current company

**Pull request merged**